test_sort.py

```diff
@@ -0,0 +1,14 @@
+""""
+Pytest tests for insertion sort from main.py
+"""
+from main import *
+
+
+def test_insertion_sort():
+    unsorted_arr = [1, 5, 7, 3, 2]
+    insertionSort(unsorted_arr)
+    assert unsorted_arr == [1, 2, 3, 5, 7]
+
+
+if __name__ == "__main__":
```

Comment:
FYI, you don't actually need to add this. Instead you can just run `pytest` and it is smart enough to know where your tests are. Otherwise, it looks great! 👍🏻

Reply:
Ok sweet good to know I will change that.